modules/tfm/tfm/boards/partition/flash_layout.h

```diff
@@ -86,9 +86,9 @@
 #endif
 
 /* Non-secure storage region */
-#if defined(PM_SETTINGS_STORAGE_ADDRESS) && defined(PM_SETTINGS_STORAGE_SIZE)
-#define NRF_FLASH_NS_STORAGE_AREA_OFFSET   (PM_SETTINGS_STORAGE_ADDRESS)
-#define NRF_FLASH_NS_STORAGE_AREA_SIZE     (PM_SETTINGS_STORAGE_SIZE)
+#if defined(PM_NONSECURE_STORAGE_ADDRESS)
```

Comment:
Dont we need the `&& defined(PM_NONSECURE_STORAGE_SIZE)` here?

Reply:
They are generated in pairs, but I guess it would not hurt.

Reply:
This was the exception, everywhere else it is only checking that the address is defined.